src/signals/IncidentMap/components/utils/get-filtered-incidents.ts

```diff
@@ -12,9 +15,21 @@ export const getFilteredIncidents = (
     .filter((filter) => filter.filterActive)
     .map((filter) => filter.slug)
 
-  const activeIncidents = incidents.filter((incident) =>
-    activeFilterSlugs.includes(incident.properties.category.parent.slug)
-  )
+  const excludeSubSlugs = filters
+    .filter((filter) => filter.subCategories)
+    .map((filter) => {
+      return filter.subCategories
+        ?.filter((sub) => !sub.filterActive)
+        .map((sub) => sub.slug)
+    })
+    .flat(1)
```

Comment:
I don't think this should be here. In the getFilterCategories function the filters that are available are set. There we should determine which filter should be available for the user as well. In this util (fet-filtered-incidents) we should filter the incidents based on the filters we get. 

A simple solution could be:

<img width="608" alt="Screenshot 2022-10-18 at 13 25 22" src="https://user-images.githubusercontent.com/46756331/196416802-ff97a576-982f-47ec-abb8-2e22490dca50.png">